packages/zoe/test/unitTests/contracts/test-callSpread-calculation.js

```diff
@@ -6,20 +6,23 @@ import '../../../exported.js';
 
 import { setup } from '../setupBasicMints.js';
 import { calculateShares } from '../../../src/contracts/callSpread/calculateShares.js';
-import { multiplyBy } from '../../../src/contractSupport/index.js';
+import {
+  ceilMultiplyBy,
+  floorMultiplyBy,
+} from '../../../src/contractSupport/index.js';
 import {
   make0Percent,
   make100Percent,
 } from '../../../src/contracts/callSpread/percent.js';
 
-function compareShareRatios(t, expected, actual, amount) {
+function compareShareRatios(t, actual, expected, amount) {
   t.deepEqual(
-    multiplyBy(amount, expected.longShare),
-    multiplyBy(amount, actual.longShare),
+    ceilMultiplyBy(amount, actual.longShare),
+    ceilMultiplyBy(amount, expected.longShare),
   );
   t.deepEqual(
-    multiplyBy(amount, expected.shortShare),
-    multiplyBy(amount, actual.shortShare),
+    floorMultiplyBy(amount, actual.shortShare),
```

Comment:
I note that the actual short computation is `total - ceilMultiplyBy(amount, actual.longShare)`.  I agree this should be the same and like that it's using floor instead to confirm that.